salalib/vgamodules/vgathroughvision.cpp

```diff
@@ -60,14 +59,14 @@ bool VGAThroughVision::run(Communicator *comm, const Options &, PointMap &map, b
                         map.getPoint(key).m_misc += 1;
 
                         // TODO: Undocumented functionality. Shows how many times a gate is passed?
-
                         if(hasGateColumn) {
                             auto iter = attributes.find(AttributeKey(key));
                             if(iter != attributes.end()) {
                                 int gate = static_cast<int>(iter->getRow().getValue(g_col_gate));
-                                if (gate != -1 && seengates.searchindex(gate) == paftl::npos) {
+                                auto gateIter = depthmapX::findBinary( seengates, gate );
```

Comment:
Probably should invert the if statements here a bit - if `gate == -1`, there is no point starting a binary search for it.